nixos/modules/services/misc/graphical-desktop.nix

```diff
@@ -20,14 +19,16 @@ in
 
     environment = {
       # localectl looks into 00-keyboard.conf
-      etc."X11/xorg.conf.d/00-keyboard.conf".text = ''
+      etc."X11/xorg.conf.d/00-keyboard.conf".text = let
```

Comment:
Weirdly, we may want to generate this even on Wayland, because localectl looks at it.

Reply:
We can probably leave it up to individual DEs’ NixOS modules.

Reply:
graphical desktop is meant to be used by xserver and wayland and as long as you have any displayManager enabled, you should already get it. Otherwise most wayland DE users should have seen regressions since it was introduced.